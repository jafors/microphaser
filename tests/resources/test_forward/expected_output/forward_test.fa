>d0e162ace9c7c27F
AAAATGATTGCAGACTTTCTGGTTGAG
>b757c2791648ea0F
AAAATGATTGCAGACTTTCTGGTTGtG
>14e579cd1a6da0fF
ATGATTGCAGACTTTCTGGTTGAGCGC
>0a2f1659db4e248F
ATGATTGCAGACTTTCTGGTTGtGCGC
>31b947f29ca3de1F
ATTGCAGACTTTCTGGTTGAGCGCAAG
>3b8fe1e116aa08dF
ATTGCAGACTTTCTGGTTGtGCGCAAG
>4982561133be9e2F
GCAGACTTTCTGGTTGAGCGCAAGACT
>7bbe4ec4d3f1fd7F
GCAGACTTTCTGGTTGtGCGCAAGACT
>43c694617025731F
GACTTTCTGGTTGAGCGCAAGACTATT
>69d1603a3619462F
GACTTTCTGGTTGtGCGCAAGACTATT
>c03086ec539e5e4F
TTTCTGGTTGAGCGCAAGACTATTTCT
>1dc9a196a389081F
TTTCTGGTTGtGCGCAAGACTATTgCT
>370d73d49de8b01F
CTGGTTGAGCGCAAGACTATTTCTTTT
>377637b77f9ca9bF
CTGGTTGtGCGCAAGACTATTgCTTTT
>0f8dcb02a921d38F
GTTGAGCGCAAGACTATTTCTTTTGAT
>c757c189c7eb526F
GTTGtGCGCAAGACTATTgCTTTTGAT
>a2370b92773f6eaF
GAGCGCAAGACTATTTCTTTTGATGCC
>cac5c40f7e2c1f6F
GtGCGCAAGACTATTgCTTTTGATGCC
>fe7deda783ad5b0F
TTCACGCTTATTTTAAACCCTGTAATC
>9e90f4c3f608a85F
TTCACGCcTATTTTAAACCCTGTAATg
>ada969ed0f4ca81F
ACGCTTATTTTAAACCCTGTAATCTAC
>57351b40eb5f98bF
ACGCcTATTTTAAACCCTGTAATgTAC
>843865ced5ae940F
CTTATTTTAAACCCTGTAATCTACACG
>0ebde5f457c5fabF
CcTATTTTAAACCCTGTAATgTACACG
>a95d7f1dcbc9644F
ATTTTAAACCCTGTAATCTACACGCTA
>334c7ac82a2151dF
ATTTTAAACCCTGTAATgTACACGCTA
>9727daff337716cF
TTAAACCCTGTAATCTACACGCTAAGA
>058cbe6badb7f95F
TTAAACCCTGTAATgTACACGCTAAGA
>a15150384715033F
AACCCTGTAATCTACACGCTAAGAAAC
>54c7ec5e0d8c468F
AACCCTGTAATgTACACGCTAAGAAAC
>89a52c037a905f2F
CCTGTAATCTACACGCTAAGAAACAAA
>8c6f6c7c66d0fe8F
CCTGTAATgTACACGCTAAGAAACAAA
>a41b9d336833ef9F
GTAATCTACACGCTAAGAAACAAAGAA
>597dd158b206912F
GTAATgTACACGCTAAGAAACAAAGAA
>4f447f9f157e0ebF
ATCTACACGCTAAGAAACAAAGAAGTG
>a71503b7a96d0e9F
ATgTACACGCTAAGAAACAAAGAAGTG
>797be35fb510426F
AAAATGATTGCAGACTTTCTGGTTGAG
>c4874ef3698e68bF
AAAATGATTGCAGACTTTCTGGTTGtG
>ff3280b1618385cF
ATGATTGCAGACTTTCTGGTTGAGCGC
>698dd360e63c4c8F
ATGATTGCAGACTTTCTGGTTGtGCGC
>46fa3bf2358f244F
ATTGCAGACTTTCTGGTTGAGCGCAAG
>496dfdab68a0492F
ATTGCAGACTTTCTGGTTGtGCGCAAG
>c53ca816247d999F
GCAGACTTTCTGGTTGAGCGCAAGACT
>370e54d8e185f8dF
GCAGACTTTCTGGTTGtGCGCAAGACT
>48a0508d1d2689aF
GACTTTCTGGTTGAGCGCAAGACTATT
>b9a6ec9b4c46ca0F
GACTTTCTGGTTGtGCGCAAGACTATT
>9c1f4877515245bF
TTTCTGGTTGAGCGCAAGACTATTTCT
>ee36603d6ff4115F
TTTCTGGTTGtGCGCAAGACTATTgCT
>ae32818621ea6c2F
CTGGTTGAGCGCAAGACTATTTCTTTT
>c42cf30c828ab39F
CTGGTTGtGCGCAAGACTATTgCTTTT
>5188e5f97c85118F
GTTGAGCGCAAGACTATTTCTTTTGAT
>950d779d00ccf03F
GTTGtGCGCAAGACTATTgCTTTTGAT
>4ff22878ccacb7bF
GAGCGCAAGACTATTTCTTTTGATGCC
>d1e204f1dae5d70F
GtGCGCAAGACTATTgCTTTTGATGCC
>3e98e8cd4dc4194F
TTCACGCTTATTTTAAACCCTGTAATC
>33adcda9340ca7aF
TTCACGCcTATTTTAAACCCTGTAATg
>81d90839cd37f79F
ACGCTTATTTTAAACCCTGTAATCTAC
>23b31b9a6c2b9e4F
ACGCcTATTTTAAACCCTGTAATgTAC
>029174d2234b684F
CTTATTTTAAACCCTGTAATCTACACG
>9db2ab03c532edaF
CcTATTTTAAACCCTGTAATgTACACG
>203918509ab64a9F
ATTTTAAACCCTGTAATCTACACGCTA
>8536199aac8252bF
ATTTTAAACCCTGTAATgTACACGCTA
>35ce88e977bf036F
TTAAACCCTGTAATCTACACGCTAAGA
>cce445ce8846127F
TTAAACCCTGTAATgTACACGCTAAGA
>94245e6123914b3F
AACCCTGTAATCTACACGCTAAGAAAC
>7625c431cf06036F
AACCCTGTAATgTACACGCTAAGAAAC
>4fb7424e01e4fbfF
CCTGTAATCTACACGCTAAGAAACAAA
>7b415a8577311faF
CCTGTAATgTACACGCTAAGAAACAAA
>98c670f729c8cc6F
GTAATCTACACGCTAAGAAACAAAGAA
>5039fb5e4d52a65F
GTAATgTACACGCTAAGAAACAAAGAA
>6c60e51b56014acF
ATCTACACGCTAAGAAACAAAGAAGTG
>b746f5c85a36d30F
ATgTACACGCTAAGAAACAAAGAAGTG
