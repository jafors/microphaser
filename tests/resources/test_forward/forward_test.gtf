chr14	ensembl_havana	gene	19975444	19976659	.	+	.	gene_id "ENSG00000169488"; gene_version "6"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding";
chr14	havana	transcript	19975444	19976659	.	+	.	gene_id "ENSG00000169488"; gene_version "6"; transcript_id "ENST00000641512"; transcript_version "1"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "OR4K15-202"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "basic";
chr14	havana	exon	19975444	19976659	.	+	.	gene_id "ENSG00000169488"; gene_version "6"; transcript_id "ENST00000641512"; transcript_version "1"; exon_number "1"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "OR4K15-202"; transcript_source "havana"; transcript_biotype "protein_coding"; exon_id "ENSE00001635862"; exon_version "1"; tag "basic";
chr14	havana	CDS	19975591	19976562	.	+	0	gene_id "ENSG00000169488"; gene_version "6"; transcript_id "ENST00000641512"; transcript_version "1"; exon_number "1"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "OR4K15-202"; transcript_source "havana"; transcript_biotype "protein_coding"; protein_id "ENSP00000493198"; protein_version "1"; tag "basic";
chr14	havana	start_codon	19975591	19975593	.	+	0	gene_id "ENSG00000169488"; gene_version "6"; transcript_id "ENST00000641512"; transcript_version "1"; exon_number "1"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "OR4K15-202"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "basic";
chr14	havana	stop_codon	19976563	19976565	.	+	0	gene_id "ENSG00000169488"; gene_version "6"; transcript_id "ENST00000641512"; transcript_version "1"; exon_number "1"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "OR4K15-202"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "basic";
chr14	havana	five_prime_utr	19975444	19975590	.	+	.	gene_id "ENSG00000169488"; gene_version "6"; transcript_id "ENST00000641512"; transcript_version "1"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "OR4K15-202"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "basic";
chr14	havana	three_prime_utr	19976566	19976659	.	+	.	gene_id "ENSG00000169488"; gene_version "6"; transcript_id "ENST00000641512"; transcript_version "1"; gene_name "OR4K15"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "OR4K15-202"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "basic";
