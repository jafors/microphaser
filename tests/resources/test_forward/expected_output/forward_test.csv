id	transcript	gene_id	gene_name	chrom	offset	freq	nvar	nsomatic	nvariant_sites	nsomvariant_sites	strand	positions	aa_change
d0e162ace9c7c27F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975827	0.5348837209302325	0	0	1	1	Forward	19975852	p.Glu112Val
b757c2791648ea0F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975827	0.46511627906976744	1	1	1	1	Forward	19975852	p.Glu112Val
14e579cd1a6da0fF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975830	0.4782608695652174	0	0	1	1	Forward	19975852	p.Glu112Val
0a2f1659db4e248F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975830	0.5217391304347826	1	1	1	1	Forward	19975852	p.Glu112Val
31b947f29ca3de1F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975833	0.42857142857142855	0	0	1	1	Forward	19975852	p.Glu112Val
3b8fe1e116aa08dF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975833	0.5714285714285714	1	1	1	1	Forward	19975852	p.Glu112Val
4982561133be9e2F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975836	0.4090909090909091	0	0	1	1	Forward	19975852	p.Glu112Val
7bbe4ec4d3f1fd7F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975836	0.5909090909090909	1	1	1	1	Forward	19975852	p.Glu112Val
43c694617025731F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975839	0.43478260869565216	0	0	1	1	Forward	19975852	p.Glu112Val
69d1603a3619462F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975839	0.5652173913043478	1	1	1	1	Forward	19975852	p.Glu112Val
c03086ec539e5e4F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975842	0.4166666666666667	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
1dc9a196a389081F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975842	0.5833333333333334	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
370d73d49de8b01F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975845	0.4166666666666667	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
377637b77f9ca9bF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975845	0.5833333333333334	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
0f8dcb02a921d38F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975848	0.44	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
c757c189c7eb526F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975848	0.56	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
a2370b92773f6eaF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975851	0.44	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
cac5c40f7e2c1f6F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19975851	0.56	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
fe7deda783ad5b0F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976421	0.52	0	0	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
9e90f4c3f608a85F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976421	0.48	2	1	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
ada969ed0f4ca81F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976424	0.5051546391752577	0	0	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
57351b40eb5f98bF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976424	0.4948453608247423	2	1	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
843865ced5ae940F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976427	0.5098039215686274	0	0	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
0ebde5f457c5fabF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976427	0.49019607843137253	2	1	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
a95d7f1dcbc9644F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976430	0.49074074074074076	0	0	1	1	Forward	19976447	p.Ile310Met
334c7ac82a2151dF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976430	0.5092592592592593	1	1	1	1	Forward	19976447	p.Ile310Met
9727daff337716cF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976433	0.5178571428571429	0	0	1	1	Forward	19976447	p.Ile310Met
058cbe6badb7f95F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976433	0.48214285714285715	1	1	1	1	Forward	19976447	p.Ile310Met
a15150384715033F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976436	0.5221238938053098	0	0	1	1	Forward	19976447	p.Ile310Met
54c7ec5e0d8c468F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976436	0.4778761061946903	1	1	1	1	Forward	19976447	p.Ile310Met
89a52c037a905f2F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976439	0.5263157894736842	0	0	1	1	Forward	19976447	p.Ile310Met
8c6f6c7c66d0fe8F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976439	0.47368421052631576	1	1	1	1	Forward	19976447	p.Ile310Met
a41b9d336833ef9F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976442	0.532258064516129	0	0	1	1	Forward	19976447	p.Ile310Met
597dd158b206912F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976442	0.46774193548387094	1	1	1	1	Forward	19976447	p.Ile310Met
4f447f9f157e0ebF	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976445	0.5522388059701493	0	0	1	1	Forward	19976447	p.Ile310Met
a71503b7a96d0e9F	ENST00000641512	ENSG00000169488	OR4K15	chr14	19976445	0.44776119402985076	1	1	1	1	Forward	19976447	p.Ile310Met
797be35fb510426F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975827	0.5348837209302325	0	0	1	1	Forward	19975852	p.Glu112Val
c4874ef3698e68bF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975827	0.46511627906976744	1	1	1	1	Forward	19975852	p.Glu112Val
ff3280b1618385cF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975830	0.4782608695652174	0	0	1	1	Forward	19975852	p.Glu112Val
698dd360e63c4c8F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975830	0.5217391304347826	1	1	1	1	Forward	19975852	p.Glu112Val
46fa3bf2358f244F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975833	0.42857142857142855	0	0	1	1	Forward	19975852	p.Glu112Val
496dfdab68a0492F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975833	0.5714285714285714	1	1	1	1	Forward	19975852	p.Glu112Val
c53ca816247d999F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975836	0.4090909090909091	0	0	1	1	Forward	19975852	p.Glu112Val
370e54d8e185f8dF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975836	0.5909090909090909	1	1	1	1	Forward	19975852	p.Glu112Val
48a0508d1d2689aF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975839	0.43478260869565216	0	0	1	1	Forward	19975852	p.Glu112Val
b9a6ec9b4c46ca0F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975839	0.5652173913043478	1	1	1	1	Forward	19975852	p.Glu112Val
9c1f4877515245bF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975842	0.4166666666666667	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
ee36603d6ff4115F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975842	0.5833333333333334	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
ae32818621ea6c2F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975845	0.4166666666666667	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
c42cf30c828ab39F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975845	0.5833333333333334	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
5188e5f97c85118F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975848	0.44	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
950d779d00ccf03F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975848	0.56	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
4ff22878ccacb7bF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975851	0.44	0	0	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
d1e204f1dae5d70F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19975851	0.56	2	1	2	1	Forward	19975852|19975866	p.Glu112Val|p.Ser117Ala
3e98e8cd4dc4194F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976421	0.52	0	0	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
33adcda9340ca7aF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976421	0.48	2	1	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
81d90839cd37f79F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976424	0.5051546391752577	0	0	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
23b31b9a6c2b9e4F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976424	0.4948453608247423	2	1	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
029174d2234b684F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976427	0.5098039215686274	0	0	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
9db2ab03c532edaF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976427	0.49019607843137253	2	1	2	1	Forward	19976428|19976447	p.Leu304Pro|p.Ile310Met
203918509ab64a9F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976430	0.49074074074074076	0	0	1	1	Forward	19976447	p.Ile310Met
8536199aac8252bF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976430	0.5092592592592593	1	1	1	1	Forward	19976447	p.Ile310Met
35ce88e977bf036F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976433	0.5178571428571429	0	0	1	1	Forward	19976447	p.Ile310Met
cce445ce8846127F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976433	0.48214285714285715	1	1	1	1	Forward	19976447	p.Ile310Met
94245e6123914b3F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976436	0.5221238938053098	0	0	1	1	Forward	19976447	p.Ile310Met
7625c431cf06036F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976436	0.4778761061946903	1	1	1	1	Forward	19976447	p.Ile310Met
4fb7424e01e4fbfF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976439	0.5263157894736842	0	0	1	1	Forward	19976447	p.Ile310Met
7b415a8577311faF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976439	0.47368421052631576	1	1	1	1	Forward	19976447	p.Ile310Met
98c670f729c8cc6F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976442	0.532258064516129	0	0	1	1	Forward	19976447	p.Ile310Met
5039fb5e4d52a65F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976442	0.46774193548387094	1	1	1	1	Forward	19976447	p.Ile310Met
6c60e51b56014acF	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976445	0.5522388059701493	0	0	1	1	Forward	19976447	p.Ile310Met
b746f5c85a36d30F	ENST00000305051	ENSG00000169488	OR4K15	chr14	19976445	0.44776119402985076	1	1	1	1	Forward	19976447	p.Ile310Met
